src/helpers/eventBridge.ts

```diff
@@ -238,4 +239,12 @@ export default class EventBridge {
 
     return true;
   }
+
+  async createObjectFromSchema(schema: any): Promise<void> {
+    const jsf = require("json-schema-faker");
```

Comment:
Why the use of `require` here? The ESLint config forbids the use of `require`, and results in the following error:

`ESLint: Require statement not part of import statement.(@typescript-eslint/no-var-requires)`